tests/Resources/Page/OCPDashboard/Monitoring/Grafana.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Documentation    Cointans Grafana Releted keyword
+Library  SeleniumLibrary
+
+
+*** Variables ***
+${SLIDE_MENU_DROPDOWN}     //div[@class="sidemenu-item dropdown"]
+${GRAFANA_SEARCH_BUTTION}        //*[text()="Search"]
+${JUPYTERHB_SLI}        //*[contains(@aria-label,"Jupyterhub SLIs")]
+
+
+*** Keywords ***
+Verify If Jupyterhub Sli Is Present
```

Comment:
As Grafana is not part of OpenShift's Console, I think this file should be in tests/Resources/Page/ODH/Grafana/Grafana.resource

`Verify If Jupyterhub Sli Is Present` is very specific to RHODS. This keyword should be in the test itself (and maybe rename it to `Verify If Jupyterhub SLIs Dashboard Is Present`). 

Grafana.resource could include these two keywords to help implementing the above keyword:
- `Go To Search`: goes to the search section and waits until is loaded
   
- `Dashboard Should Exist      $dashboard_name`:   Verifies that dashboard $dashboard_name is listed in Search section